content/about/faq.md

```diff
@@ -1,23 +1,54 @@
 ---
-title: FAQ
+title: FAQs
 weight: 1200
 type: essay
 online: false
 ---
 
-**Why is it called Quire?**
+### What is Eleventy (11ty)?
 
-From *Understanding Illuminated Manuscripts: A Guide to Technical Terms (Getty Publications)*:
-> *Quires are the “gatherings” or “booklets” of which a book is formed. Quire numeration, which began in the Late Antique period, consists of numbers written on a quire (usually on its final verso) to facilitate arrangement during binding. Quire signatures (or leaf signatures) are numbers and/or letters written in a quire to facilitate the arrangement of its internal components. These were at first ad hoc, but beginning around 1400 they might follow a system: for example, ai-aiv could be written on the first four leaves of the first quire; bi-biv on the first four leaves of the second quire, and so on. Such annotations only became widespread from the late thirteenth century on.*
+Eleventy (11ty) is a simple yet powerful [static site generator](https://quire.getty.edu/about/how-it-works/) based on JavaScript. It was created as an alternative to [Jekyll](https://jekyllrb.com/) and will replace our former static site generator, [Hugo](https://gohugo.io/). 11ty transforms a directory of various templates into HTML and is popular for its ability to mix multiple templating languages (including: HTML, Markdown, JavaScript, Liquid, Nunjucks, Handlebars, Mustache, EJS, Haml, Pug, and Custom). It is available as open source under the MIT License. Read more on the [11ty website](https://www.11ty.dev/).
 
-From Google:
-> *quire: 1. four sheets of paper or parchment folded to form eight leaves, as in medieval manuscripts. 2. any collection of leaves one within another in a manuscript or book. 3. 25 (formerly 24) sheets of paper; one-twentieth of a ream.*
+### What are the benefits of 11ty?
 
-In Latin:
-> *Quire is also a form of the Latin verb “queo”, or “I am able” or “I can”. Specifically, it’s the 2nd principal part and so means “to be able”. It’s also intransitive and expects an action. To be able to write, “quire scribere”. To be able to publish, “quire prodere”. To be able to edit, “quire corrigere”. Quire literally “can” do anything.*
+11ty has a significantly faster processing time than other static site generators. With the move to 11ty, we have reworked the overall file structure of Quire projects to simplify navigation and make essential project files more accessible. On the development side, one of 11ty's great strengths is its ability to mix templating languages. For example, in Quire projects, Liquid will be used to generate simple markup while JavaScript will be used where data manipulation is required (more on those two langauges below.) Other templating languages may also be used. Above all, the move to 11ty will enable the expansion of advanced features, a more robust set of CLI options, and improve developer support and contribution opportunities.
```

Comment:
Hugo is actually also very fast, fyi. It's all the other stuff that slows things down. That will likely be the same with us and 11ty.